.prow.yaml

```
@@ -10,4 +10,25 @@ presubmits:
       - image: ghcr.io/kcp-dev/infra/build:1.21.8-1
         command:
         - make
-        - test
\ No newline at end of file
+        - test
+
+  - name: pull-kcp-test-e2e
+    decorate: true
+    # only run e2e tests if code changed.
+    run_if_changed: "(cmd|config|pkg|sdk|test|go.mod|go.sum|Makefile|.prow.yaml)"
```

This should probably be something like

```suggestion
    run_if_changed: "(pkg|examples|go.mod|go.sum|Makefile|.prow.yaml)"
```